jgiven-spring/src/main/java/com/tngtech/jgiven/integration/spring/JGivenStage.java

```diff
@@ -0,0 +1,29 @@
+package com.tngtech.jgiven.integration.spring;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.stereotype.Component;
+
+/**
+ * Annotation that marks a bean as JGiven Stage.
+ *
+ * @since 0.8.0
+ */
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Component
+public @interface JGivenStage {
```

Comment:
`@JGivenStage` is meant as replacment for `@Component` which also does not use `@Inherited`